Makefile

```diff
@@ -13,7 +13,16 @@ run:
 	echo "-server is running"
 
 format:
-	black .
+	black . --exclude '/alembic/'
 	echo "-code is formatted with black"
-	ruff . --fix
+	ruff . --exclude '/alembic/'
 	echo "-code is formatted with ruff"
+
+migration:
+	alembic revision --autogenerate -m "Create table"
```

Comment:
Remove this command please,migration names have to have different names depending on what the migrations is about, we cannot use the same name for all migrations